ui/src/app/job-list/job-list.component.ts

```diff
@@ -71,9 +73,9 @@ export class JobListComponent implements OnInit {
     this.projectId = req.extensions.projectId || '';
 
     this.header.pageSubject.subscribe(resp => this.onClientPaginate(resp));
-    const savedPageSettings = this.settingsService.getPageSize(this.projectId);
-    if (savedPageSettings) {
-      this.pageSize = savedPageSettings;
+    this.savedProjectSettings = this.settingsService.getSettingsForProject(this.projectId);
```

Comment:
Hmm, I think I probably prefer calling settingsService each time. I would worry about this local copy of the settings going out of sync over time.

Reply:
ok – changed